src/nifgen/metadata/nifgen.proto

```diff
@@ -1992,7 +1992,7 @@ message SetAttributeViStringRequest {
   string channel_name = 2;
   NiFgenAttribute attribute_id = 3;
   oneof attribute_value_enum {
-    string attribute_value_raw = 4;
+    string attribute_value = 4;
```

Comment:
I'm testing this out to see if it fixes the build (it did locally), but if the proto does need to be updated then there's a hapigen change I should make before submitting this.

Reply:
It did fix the build, but since the proto file in the latest export of nifgen_grpc_device is identical, it seems like this is still an issue that I have to sort out in nimi-python.

Reply:
Fixed this so that both the fgen and dcpower tests pass without modifying any proto files.